backstory/src/main/webapp/gpt2.js

```diff
@@ -90,3 +93,37 @@ function formatResponse(text) {
 
   return container;
 }
+
+// CREATE BACKSTORY LOADING ELEMENT
+
+/**
+ * Helper function to create and set a loading element to display after the
+ * photo-upload form is submitted, while image is being analyzed and the
+ * backstory is being created. Only execute this function if valid
+ * image uploaded.
+ */
+function createBackstoryLoadingElement() {
```

Comment:
Is there any way to export this function since we already use it in the main script?